packages/contracts-bedrock/test/L2/L1BlockInterop.t.sol

```diff
@@ -202,3 +203,110 @@ contract L1BlockInteropTest is CommonTest {
         return L1BlockInterop(address(l1Block));
     }
 }
+
+contract L1BlockInteropIsDeposit_Test is L1BlockInteropTest {
```

Comment:
I chose to create the test contract for each function just as other test files, even though they are not following that styling on this one. Lmk wdyt as well @skeletor-spaceman